scripts/build.sh

```diff
@@ -2,8 +2,26 @@
 echo -e "\e[44;97m Compiling ... \e[0m"
 if GOOS=linux GOBIN=`pwd`/bin go install -v ./cmd/appgw-ingress; then
     chmod -R 777 bin
-    echo -e "\e[42;97m SUCCESS \e[0m"
+    echo -e "\e[42;97m BUILD SUCCESS \e[0m"
 else
-    echo -e "\e[101;97m FAILED \e[0m"
+    echo -e "\e[101;97m BUILD FAILED \e[0m"
     exit 1
 fi
+
+echo -e "\e[44;97m Running go lint ... \e[0m"
```

Comment:
We should run the linter before the compilation? Linters are generally faster. The compilations would take longer. So we can potentially error out earlier.

Reply:
Will do.